doc/_templates/layout.html

```diff
@@ -7,4 +7,5 @@
     </div>
   {% endif %}
   {{ super() }}
+<a href="https://github.com/zephyrproject-rtos/zephyr"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png" alt="Fork me on GitHub"></a>
```

Comment:
I don't like this corner banner on every page.  Seems distracting and unnecessary.

Reply:
we can have it on the top page only if that helps.